test/run_test.py

```diff
@@ -14,7 +14,7 @@
 
 #easy run
 sys.stderr.write("Running database creation + DBSCAN model fit + fit refinement (--easy-run)\n")
-subprocess.run("python ../poppunk-runner.py --easy-run --r-files references.txt --min-k 13 --k-step 3 --output example_db --full-db --overwrite", shell=True, check=True)
+subprocess.run("python ../poppunk-runner.py --easy-run --r-files references.txt --min-k 13 --k-step 3 --output example_db --full-db --qc-filter prune --overwrite", shell=True, check=True)
```

Comment:
Add a test with `continue` too?

Reply:
Added in b039242, also tests the length range approach to filtering assemblies.